feature-configs/cn-ran-overlays/ran-profile/is_ready.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Check for updated worker-cnf node
+
+oc wait mcp/worker-cnf --for condition=updated --timeout 1s
```

Comment:
that assumes the mcp name is worker-cnf.
we are already kustomizing that to something else (worker-du) in some cases.
and changing the script for that would be difficult.
I suggest adding an environment variable

so 
```
oc wait ${CNF_MCP} --for condition ...
```

I'll actually open a separate PR about it, because the fix need to be cross the whole repo

Reply:
I've changed this to use ${CNF_MCP:-"mcp/worker-cnf"}, i.e. to use the environment variable you recommended, with a compatibility default to worker-cnf if unset.